lib/helpers/route_page.dart

```diff
@@ -8,14 +8,13 @@ import 'package:retro_shopping/views/home.dart';
 import 'package:retro_shopping/views/profile.dart';
 import 'package:retro_shopping/views/search.dart';
 import 'package:retro_shopping/widgets/product/product_page.dart';
-import 'package:retro_shopping/widgets/splash.dart';
 
 class RoutePage {
   static Route<dynamic> generateRoute(RouteSettings settings) {
     switch (settings.name) {
       case '/':
```

Comment:
Remove this case, as we already have a case for Dashboard.

Reply:
I've made the case '/' point to dashboard as well.
And doing the squash now

Reply:
@AJAYK-01 there already exists a case for dashboard, so you can remove this case